playPage.html

```diff
@@ -0,0 +1,136 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <!-- Material Icons -->
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet" />
+    <!-- the CSS file --> 
+    <link rel="stylesheet" href="style.css" />
+    <title>YouTube clone with HTML & CSS</title>
+</head>
+
+
+<body>
+    <header class="header">
+        <div class="logo left">
+            <i id="menu" class="material-icons">menu</i>
+            <img src="https://www.freecodecamp.org/news/content/images/2022/01/yt-logo.png">
+        </div>
+
+        <div class="search center">
+            <form action="">
+                <input type="text" placeholder="Search" />
+                <button><i class="material-icons">search</i> </button>
+            </form>
+            <i class="material-icons mic">mic</i>
+        </div>
+
+        <div class="icons right">
+            <i class="material-icons">videocam</i>
+            <i class="material-icons">apps</i>
+            <i class="material-icons">notifications</i>
+            <i class="material-icons display-this">account_circle</i>
+        </div>
+
+    </header>
+    <main>
+
+        <div class="content">
+            
+            <div class="video-container">
+
+                <iframe width="720" height="576"
+                    src="https://www.youtube.com/embed/ObAcg1k25YI">
+                </iframe>
+                <div class="title">
+                    <h3>
+                        Home — Another Kid | Free Background Music | Audio Library Release
+                    </h3>
+                    <a href="">
+                        Audio Library Plus — Official Releases
+
+                    </a>
+                    <span> 32K Views • 2 weeks ago </span>
+                </div>
+            </div>
+                
+            
+        </div>
+        <div class="suggestions">
```

Comment:
I suggest that you should not add blank lines, spaces, or indentations without a reason. One blank line is enough

Reply:
- [ ] Although the reason may seem mundane and obvious you should always provide a good reason (a "why") for your requested changes as the whole purpose of a code review is to help the other person **learn** from their previous mistakes. You could for example say "...that would improve the HTML structure and make the code base easily accessible" this would definitely help. 💪
- [ ] [OPTIONAL] It would be more friendly and less serious if you could use a couple of emojis and/or gifs in the inline comments. 😊